src/Illuminate/Foundation/Auth/AuthenticatesUsers.php

```diff
@@ -127,7 +127,7 @@ protected function sendFailedLoginResponse(Request $request)
         $errors = [$this->username() => trans('auth.failed')];
 
         if ($request->expectsJson()) {
-            return response()->json($errors, 422);
+            return response()->json(['message' => 'Authentication failed.'], 422);
```

Comment:
I think we would still want the message to be `trans('auth.failed')` and the `$errors` variable is now unused.